src/test/java/hw2/ex2/Ex2.java

```diff
@@ -0,0 +1,65 @@
+package hw2.ex2;
+
+import hw2.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.Test;
+
+public class Ex2 extends BaseTest {
+
+  @Test
+  public void exerciseTwoTest() {
+    //1 Open test site by URL
+    openPage();
+
+    //2 Assert Browser title
+    softAssert.assertEquals(driver.getTitle(), "Home Page", "Invalid title on the page");
```

Comment:
Why do you decide use `softAssertion` for the Ex2?

Reply:
I considered it's still suitable to use soft asserts there because of the big amount of UI verifications. So in the case of any changes in the UI, these tests could provide a more detailed report.

Reply:
yeap, good point. However, for the second task required use hard asserts

Reply:
ok, got it. now it's fixed